app/frontend/public/stylesheets/recipe.css

```diff
@@ -44,10 +44,64 @@ h1 {
 
 img {
   display: block;
-  max-width: 100%;
+  width: 100%;
   height: auto;
 }
 
+.tags {
+  margin: 0 auto 2rem auto;
+  padding-left: 2rem;
+  max-width: 68rem;
+}
+
+.tags span:before {
+  content: "";
+  position: absolute;
```

Comment:
We may not be allowed absolute? Don't know enough about the component though so i trust you if you think it's fine